pkgs/by-name/an/ansel/package.nix

```diff
@@ -160,7 +160,10 @@ stdenv.mkDerivation {
     )
   '';
 
-  passthru.updateScript = unstableGitUpdater { };
+  passthru.updateScript = unstableGitUpdater {
+    # Tags inherited from Darktable, + a "nightly" 0.0.0 tag that new artefacts get attached to
```

Comment:
```suggestion
    # Tags inherited from Darktable, + a "nightly" 0.0.0 tag that new artifacts get attached to
```

Reply:
AE vs BE difference, but sure can change it.

Reply:
this project has historically leaned BrE, and anyway it's not a typo